snapshots/overlay/overlay_test.go

```diff
@@ -173,12 +173,21 @@ func testOverlayOverlayMount(t *testing.T, newSnapshotter testsuite.SnapshotterF
 		upper = "upperdir=" + filepath.Join(bp, "fs")
 		lower = "lowerdir=" + getParents(ctx, o, root, "/tmp/layer2")[0]
 	)
-	for i, v := range []string{
+
+	expected := []string{
 		"index=off",
+	}
+	if userxattr, err := NeedsUserXAttr(root); err != nil {
```

Comment:
could you mind to do it in sync.Once?

The rest looks good to me.

Reply:
The function depends on the value of `root` string. Not sure how we can effectively use sync.Once.